torch/serialization.py

```diff
@@ -606,7 +608,14 @@ def persistent_load(saved_id):
     _sys_info = pickle_module.load(f, **pickle_load_args)
     unpickler = pickle_module.Unpickler(f, **pickle_load_args)
```

Comment:
From discussions in #21743 it's probably fine to change the default encoding here to `utf-8` and remove the `try-catch` below since that shouldn't be an issue anymore

Reply:
Thank you for the review! My only suggestion is to keep the `try-except` block even though `utf-8` is default. Say the user specifies `ascii` encoding and they try to load a module with `utf-8` in the docstring, they would still get the weird error like now. So my proposition is to keep the try-except __and__ set the `utf-8` to be default. I know this is really an edge case and it's probably never going to happen, but it doesn't hurt to keep the error handling part. :) 

Let me know what you think and I will send the corrected code.

Thank you again for the help!

Reply:
@driazati any update on this?

Reply:
I think it's OK if you want to preserve the try-catch, but you are going to have to be super careful about the wording of the error message. The current wording is just not applicable to cases when an error would be thrown! I think it might be easiest to just remove the try-catch and if a new error case comes up, we know what specific case to write the error message for.

Reply:
I understand. Going to remove it then and update the PR. Thanks!